install/README.md

```diff
@@ -0,0 +1,22 @@
+# Script for complete Pravega Installation
+
+The purpose of this script is to sequentially install all the dependencies (i.e. Operators, Zookeeper and Bookkeeper) and Pravega Components in the right order.
+
+## Prerequisites
+
+  - Kubernetes 1.10+ with Beta APIs
+  - Helm 2.10+
+  - [Tier 2 Setup](https://github.com/pravega/pravega-operator#set-up-tier-2-storage)
+  - Copy the necessary charts to the right location
+
+To copy the charts from [Zookeeper Operator](https://github.com/pravega/zookeeper-operator/tree/master/charts) and [Bookkeeper Operator](https://github.com/pravega/bookkeeper-operator/tree/master/charts) repositories inside the charts directory of this repository.
```

Comment:
You need to mention that they will first need to git clone zk-operator and bk-operator repos and copy charts folder from there into xxx location. Please provide list of commands to be entered by users instead of explaining in verbose.